inc/opengraph/class-altis-opengraph-author-presenter.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Opengraph Meta Tag additions
+ *
+ * @package altis/seo
+ */
+
+namespace Altis\SEO\Opengraph;
+
+use Yoast\WP\SEO\Presenters\Abstract_Indexable_Tag_Presenter;
+
+/**
+ * Adds a custom Opengraph Article Author meta tag.
+ */
+class Altis_Opengraph_Author_Presenter extends Abstract_Indexable_Tag_Presenter {
```

Comment:
We're already in both `Altis` and `Opengraph` namespaces so I think just `Author_Presenter` would be fine and remove the redundancy. Same for the other classes.